package.json

```diff
@@ -34,6 +34,7 @@
     "test:types": "yarn foreach test:types"
   },
   "resolutions": {
+    "@metamask/snaps-sdk@^6.24.0": "npm:@metamask/snaps-sdk@^7.0.0",
```

Comment:
This one is required because the latest `@metamask/snaps-utils` is using this version and not the `@metamask/snaps-sdk@^7.0.0` version.

An alternative would be to not use the `@metamask/snaps-sdk@^7.0.0` ourselves, but I thought it made sense to use it here, given that this change is "associated" with this new platform version (=== Snaps SDK version).